lib/_tls_wrap.js

```diff
@@ -492,6 +534,7 @@ function TLSSocket(socket, opts) {
   this._controlReleased = false;
   this.secureConnecting = true;
   this._SNICallback = null;
+  this._ALPNCallback = null;
```

Comment:
If this is meant to be a private field, please use a non-exported symbol instead of a `_`-prefixed field.